catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/MysqlTablePropertiesMetadata.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.mysql;
+
+import static com.datastrato.gravitino.catalog.PropertyEntry.enumImmutablePropertyEntry;
+import static com.datastrato.gravitino.catalog.PropertyEntry.integerOptionalPropertyEntry;
+import static com.datastrato.gravitino.catalog.PropertyEntry.stringOptionalPropertyEntry;
+import static com.datastrato.gravitino.catalog.PropertyEntry.stringReservedPropertyEntry;
+
+import com.datastrato.gravitino.catalog.PropertyEntry;
+import com.datastrato.gravitino.catalog.jdbc.JdbcTablePropertiesMetadata;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class MysqlTablePropertiesMetadata extends JdbcTablePropertiesMetadata {
+  public static final String ENGINE_KEY = "ENGINE";
+  public static final String DEFAULT_CHARSET_KEY = "DEFAULT CHARSET";
```

Comment:
I remeber the property name managed by Gravitino should use '-' to combine words, like `default-charset`